src/example/stokes_flow/stokes_flow.jl

```diff
@@ -0,0 +1,340 @@
+    Wd = Bcube.assemble_dirichlet_vector(U, V, mesh)
+
+    # Apply lift
+    L = L - A * Wd
+
+    # Apply homogeneous dirichlet condition
+    Bcube.apply_homogeneous_dirichlet_to_vector!(L, U, V, mesh)
+    Bcube.apply_dirichlet_to_matrix!(A, U, V, mesh)
+
+    # Solve linear system
+    sol = A \ L
+
+    sol = sol .+ Wd
+
+    ϕ = FEFunction(V, sol)
+
+    # Write solution
+    velocity, pressure = ϕ
+
+    vars = Dict("Velocity" => velocity, "Pressure" => pressure)
+    Bcube.write_vtk_lagrange(joinpath(outputpath, "output_steady"), vars, mesh)
+end
+# The obtained solution captures the Moffat vortices topology of the flow
+# ![](../assets/Stokes_flow_Moffat_vortices.png)
+
+# # Unsteady case - oscillating plate
+# ## Theory
+# Let's now consider a Stokes flow over an oscillating flat plate of length $0.1m$ and height $0.5m$. An oscillating velocity $(\sin(2\pi f t),0)$ is imposed on the bottom boundary as well as zero pressure. 
+# On all the other boundaries a "do nothing" condition is applied: $-pn + \mu \frac{\partial u}{\partial n} = 0$. 
+# The set of equations are:
+# ```math
+# \begin{cases}
+#   \rho \frac{\partial u}{\partial t} = \nabla \cdot \left( -p \mathbb{I} + \mu \nabla u \right) + f  \textrm{  in } \Omega \\
+#   \nabla \cdot u = 0 \textrm{  in } \Omega  \\
+#   u = (0,0)  \textrm{  on } \Gamma_L \cup \Gamma_R \\
+#   u = (1,0) \textrm{ and } p=0 \textrm{  on } \Gamma_T
+# \end{cases}
+# ```
+# The weak form of this problem is given by:
+# ```math
+# \begin{cases}
+#   \textrm{ Find } (u,p) \in W=V \times Q  \textrm{ such that } \forall (v,q) \in W\\
+#   m((u,p),(v,q))+a((u,p),(v,q)) = l((v,q))
+# \end{cases}
+# ``` 
+# where $m((u,p),(v,q)) = \int_\Omega \rho u \cdot v \, dx$ and
+# $l((v,q))=\int_\Omega f \cdot v \, dx$
+# $\mathbb{P2}-\mathbb{P1}$ (Taylor-Hood) elements are used to discretize the weak form of the problem which leads to the linear system:
+# ```math
+# \begin{bmatrix}
+#  M & 0 \\
+#  0 & 0 
+# \end{bmatrix} 
+# \begin{bmatrix} 
+# \dot{U} \\
+# \dot{P}
+# \end{bmatrix} +
+# \begin{bmatrix}
+#  A & B^T \\
+#  B & 0 
+# \end{bmatrix} 
+# \begin{bmatrix} 
+# U \\
+# P
+# \end{bmatrix}
+# = L
+# ``` 
+#
+# ## Commented code
+
+# Reference solution for unsteady case (Panton, R. L. (2013). Incompressible Flow, 4th edition, pages 228-236)
+# This solution is valid for a semi-infinite domain.  
+function reference_solution(t, x)
+    Uₛ = exp(-k * x[2]) * sin(2.0 * π * f * t - k * x[2])
+    T = 2.0 * π * f * t
+    Y = x[2] / sqrt(ν / (2.0 * π * f))
+    C = 1.0 - im
+    Uₜ = imag(
+        0.5 *
+        exp(-im * T - C * Y / sqrt(2)) *
+        erfc(sqrt(0.5 * T) * (C - Y / (T * sqrt(2)))) -
+        0.5 *
+        exp(-im * T + C * Y / sqrt(2)) *
+        erfc(sqrt(0.5 * T) * (C + Y / (T * sqrt(2)))),
+    )
+    return Uₛ + Uₜ
+end
+
+# Function that runs the unsteady case:
+function run_unsteady()
+    # Read 2D mesh
+    mesh_path = joinpath(outputpath, "mesh.msh")
+    gen_rectangle_mesh(
```

Comment:
If you use a quad mesh, why not using `rectangle_mesh` from Bcube? If the idea is to easily allow a switch to triangles, it's ok.